pagefaultlatency/README.md

```diff
@@ -0,0 +1,51 @@
+# pagefaultlatency
+
+Pagefaults are a common exception in Linux kernel. Their handling efficiency
+significantly impacts the performance of programs with frequent memory accesses.
+This tool helps assess pagefault latency when executing specific programs.
+
+`pagefaultlatency.bt` traces pagefaults by instrumenting `kprobe:handle_mm_fault`.
+Its output shows:
+- The number of pagefaults generated and the latency of each pagefault
+- The average pagefault latency
+- A pagefault latency histogram
+
+USAGE: `pagefaultlatency.bt [cmd]`
```

Comment:
Nit: should this be more like `pagefaultlatency.bt thread_name` ?

Reply:
Yes, in that sense it is, I just want to express that it is specified by the user, then I will modify it.